nightingale_dispatcher/src/nightingale_dispatcher/move_arm_task.py

```diff
@@ -9,8 +9,20 @@
 class MoveArmTask(Task):
     def __init__(self):
         self.manipulation = ManipulationControl()
+        if not self.manipulation.home_left() or not self.manipulation.home_right():
```

Comment:
- [ ] home_left should close the left gripper. the left gripper is opened by kinova